Controller/ContentTypeController.php

```diff
@@ -216,6 +233,25 @@ public function viewContentTypeAction($contentTypeId, $languageCode = null)
         $contentCount = $this->searchService->findContent($countQuery, [], false)->totalCount;
         $deleteForm = $this->createForm(new ContentTypeDeleteType(), ['contentTypeId' => $contentTypeId]);
 
+        if (isset($languageCode) && !isset($contentType->names[$languageCode])) {
+            $this->notify(
```

Comment:
This notification shows up not on the rendered page, but on the next pageload. Do you know what's wrong @dpobel?